roles/lib_openshift/src/test/unit/test_oc_adm_registry.py

```diff
@@ -245,6 +246,11 @@ def test_state_present(self, mock_cmd, mock_tmpfile_copy, mock_write):
             '/tmp/mocked_kubeconfig',
         ]
 
+        mock_oc_binary.side_effect = [
```

Comment:
For these `mock_oc_binary`s I think it's safer to simply assign `oc` to their `return_value` property. Can you swap these to use `return_value` instead? That will save us from any pain in the future if the code base changes again and the number of calls changes. `return_value` will always return just that one value. `side_effect` on an iterator will run out of values (as we saw before).

```python
    mock_oc_binary.return_value = 'oc'
```

`side_effect` is good at what it does, returning items from an iterable or *anything* from a function, but it's a little too much for this use-case.